### PR TITLE
Feat: Redis-status-check integration

### PR DESCRIPTION
**Pull request by luigibarbato**
This PR allows the integration with the `redis-status-check`. 
Now the controller is able to recognize and create the right resources for that check.

### CODE REVIEW

File: scripts/e2e/tests.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ info(){
   echo -e "${BATS_TEST_NUMBER}: ${BATS_TEST_DESCRIPTION}" >&3
 }
 
-@test "Requirements (nginx)" {
```
Comment (luigibarbato) [inline scripts/e2e/tests.sh:11]: We have to split the different tests

File: internal/checks/redis_templates.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Group | ToLower  }}-{{ .Target | ToLower  }}-redis-status-check
+  namespace: {{ .Namespace }}
+  labels:
+    app.kubernetes.io/created-by: healthcheck-controller
+    kfip.sighup.io/group: {{ .Group }}
+    kfip.sighup.io/target: {{ .Target }}
+    kfip.sighup.io/check: redis-status-check
```
Comment (angelbarrera92) [inline internal/checks/redis_templates.go:17]: `redis-status-check` or `redis` in the `kfip.sighup.io/check` label?

File: internal/checks/redis_templates.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package checks
+
+const redisManifestTemplateForServiceAccount = `
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Group | ToLower  }}-{{ .Target | ToLower  }}-redis-status-check
```
Comment (angelbarrera92) [inline internal/checks/redis_templates.go:11]: https://github.com/sighupio/fip-healthcheck-controller/pull/3/files#r723089914

File: internal/checks/redis_templates.go [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    kfip.sighup.io/group: {{ .Group }}
+    kfip.sighup.io/target: {{ .Target }}
+    kfip.sighup.io/check: redis-status-check
+`
+
+const redisManifestTemplateForRole = `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Group | ToLower  }}-{{ .Target | ToLower  }}-redis-status-check
```
Comment (angelbarrera92) [inline internal/checks/redis_templates.go:24]: https://github.com/sighupio/fip-healthcheck-controller/pull/3/files#r723089914

File: internal/checks/redis_templates.go [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  - get
+  - watch
+  - list
+`
+
+const redisManifestTemplateForRoleBinding = `
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ .Group | ToLower  }}-{{ .Target | ToLower  }}-redis-status-check
```
Comment (angelbarrera92) [inline internal/checks/redis_templates.go:48]: https://github.com/sighupio/fip-healthcheck-controller/pull/3/files#r723089914

File: internal/checks/redis_templates.go (hunk 2 above)
Comment (luigibarbato) [reply internal/checks/redis_templates.go:17]: Yeah, is `redis` according to new "policies" in the[ Notion Docs](https://www.notion.so/Health-Check-Controller-HCC-f71dc494385c4a1a8802ac432f7db8b4#01e845a1f1eb438e9ee6a85ce18b81d3).

**Review by angelbarrera92 — APPROVED** [review]
It looks really good! You catched exactly the architecture I've implemented :)

**Pull request merged**